apps/dfda-node/app/(protected)/provider/form-management/create/components/create-form-wizard.tsx

```
@@ -0,0 +1,648 @@
+    // Simulate API call
+    setTimeout(() => {
+      setIsSubmitting(false)
+      setIsSuccess(true)
+    }, 1500)
+  }
+
+  // Helper function to render question preview
+  const renderQuestionPreview = (question: FormQuestionFE) => {
+    switch (question.type) {
+      case "text":
+        const textOptions = question.options as TextQuestionOptions | null;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)